bin/docker-compose

```diff
@@ -62,7 +62,7 @@ function __main__() {
   # Build up the flags to pass to docker-compose
   local project_name="${PROJECT_NAME:-overleaf}"
 
-  local image_name="sharelatex/sharelatex"
+  local image_name="${IMAGE_NAME:-sharelatex/sharelatex}"
```

Comment:
`image_name` is overriden in the next line when `SERVER_PRO` is `true`. 

A better option perhaps is keeping `IMAGE_NAME` commented out in `overleaf.rc`, and check the existence of `IMAGE_NAME` as the last override.